### PR TITLE
style fixing

### PR DESCRIPTION
**Pull request by RomiKMoveo**
### Description
fixed stye in my app 

### Changes Made
change scss in header, Lise and detail components

I have:
- [ ] deleted all irrelevant comments, logs and debuggers.
- [ ] assure that I don't print full error stacks, full body response or headers from the server in the console.
- [ ] deleted all sensitive information from the code because it's being imported from environment variables.
- [ ] encrypted keys/passwords that are sent through the internet.
- [ ] sanitized all user inputs to prevent attacks.
- [ ] made sure my code is generic and reusable for future things or reused existing resources.
- [ ] made sure that my code is readable.
- [ ] checked that the functions and resource names are self-explanatory.
- [ ] tested the code locally and it works as expected.
- [ ] added validations and error handling in relevant cases (external requests, integrations, etc).

### CODE REVIEW

File: pokemon-minisite/src/app/components/header/header.component.scss [hunk 1]
```diff
@@ -1,24 +1,18 @@
...
-    color: var(--primary-color);
-    
-    height: 150px;
-    display: flex;
-    align-items: center;
-
+@use '../../style/variables' as *;
 
+.header {
+  color: #ffc81a;
```
Comment (sivanbasa93) [inline pokemon-minisite/src/app/components/header/header.component.scss]: please use variable

File: pokemon-minisite/src/app/components/pokemon-list/pokemon-list.component.html [hunk 2]
```diff
@@ -1,13 +1,16 @@
-<div>
-    <select class="select-dropdown" [(ngModel)]="selectedType">
-        <option value="all">All Types</option>
-        <option *ngFor="let type of types" [value]="type">{{ type }}</option>
-    </select>
-</div>
-<div>
-    <div><input [(ngModel)]="searchInput" (keydown.enter)="onSearchPokemon(searchInput)" placeholder="Search for Pokémon"></div>
+<div class="search">
+    <div>
```
Comment (sivanbasa93) [inline pokemon-minisite/src/app/components/pokemon-list/pokemon-list.component.html:3]: there are a lot of empty div , try use minimum div if there is no any effect

File: pokemon-minisite/src/app/components/pokemon-list/pokemon-list.component.scss [hunk 3]
```diff
@@ -19,6 +20,12 @@
   margin-top: 10px;
   margin-bottom: 10px;
   overflow: hidden;
+  
+  &:hover {
+    background-color: white;
+    color: #ef5450;
```
Comment (sivanbasa93) [inline pokemon-minisite/src/app/components/pokemon-list/pokemon-list.component.scss]: please use variable

File: pokemon-minisite/src/app/components/header/header.component.scss (hunk 1 above)
Comment (RomiKMoveo) [reply pokemon-minisite/src/app/components/header/header.component.scss]: done

File: pokemon-minisite/src/app/components/pokemon-list/pokemon-list.component.html (hunk 2 above)
Comment (RomiKMoveo) [reply pokemon-minisite/src/app/components/pokemon-list/pokemon-list.component.html:3]: done

File: pokemon-minisite/src/app/components/pokemon-list/pokemon-list.component.scss (hunk 3 above)
Comment (RomiKMoveo) [reply pokemon-minisite/src/app/components/pokemon-list/pokemon-list.component.scss]: done

**Pull request merged**